nnvm/python/nnvm/top/nn.py

```diff
@@ -73,6 +73,9 @@ def schedule_dense(_, outs, target):
 
 reg.register_pattern("dense", OpPattern.OUT_ELEMWISE_FUSABLE)
 
+#matmul
+reg.register_pattern("matmul", OpPattern.INJECTIVE)
```

Comment:
Matmul is unlikely going to be injective, it should be ```OUT_ELEMWISE_FUSABLE```, but the current set of schedule does not support this well.  Consider change it to OUT_ELEMWISE_FUSABLE for now and use schedule injective